deployment/roles/toy.database/tasks/main.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Install PostgreSQL
+  apt: pkg="{{ item }}" state=present
+  with_items:
```

Comment:
This is throwing a deprecation warning for me -- you don't need to use `with_items` but can instead supply a list to `pkg` directly, e.g.: https://github.com/azavea/cac-tripplanner/blob/develop/deployment/ansible/roles/cac-tripplanner.database/tasks/install-postgis-from-source.yml#L2